test/Test/Oscoin/Data/OscoinTx.hs

```diff
@@ -103,7 +103,8 @@ propTransferBalance Dict = property $ do
 
 propMultiTransfer :: forall c. Dict (IsCrypto c) -> Property
 propMultiTransfer Dict = property $ do
-    (alice, bob)  <- publicKeyPair @c
+    (alice, bob, eve) <- publicKeyTriple @c
```

Comment:
Out-loud comment: it's quite a bummer in recent version of GHCs we cannot use (easily) the (unsafe) partial pattern match without having the monad we are in satisfy `MonadFail`. In particular, creating all these `publicKeyPairXX` functions feels wasteful, when one could have had:

```haskell
publicKeys :: forall c. (IsCrypto c) => PropertyT IO [PublicKey c]
```

and later:

```haskell
[alice,bob,eve] <- take 3 <$> publicKeys
```

Reply:
Actually @geigerzaehler added `MonadFail` to hedghehog's `Property`, so we should be able to do that pretty soon! (well, when Stackage updates, which might be a while)